src/cron/dailyDigest.ts

```diff
@@ -18,9 +19,11 @@ const cron: Cron = {
   handler: async (con, logger) => {
     const personalizedDigestQuery = con
       .createQueryBuilder()
+      .select('upd.*, u.timezone')
       .from(UserPersonalizedDigest, 'upd')
+      .leftJoin(User, 'u', 'u.id = upd."userId"')
       .where(
-        'clamp_to_hours("preferredHour" - EXTRACT(HOUR FROM NOW() AT TIME ZONE "preferredTimezone")) = :preferredHourOffset',
+        `clamp_to_hours("preferredHour" - EXTRACT(HOUR FROM NOW() AT TIME ZONE COALESCE(NULLIF(u.timezone, ''), 'Etc/UTC'))) = :preferredHourOffset`,
```

Comment:
Intended to change this to use backtick?

Reply:
yes, due to `''` in query.

Reply:
Yes I can also escape but this is more readable in my opinion.

Reply:
Ah, didn't catch the `''`, I was only looking for `${}`